rero_ils/modules/patron_transaction_events/sample_data/sample-scenario-overdue1.json

```diff
@@ -0,0 +1,100 @@
+[
+  {
+    "$schema": "https://ils.rero.ch/schema/patron_transaction_events/patron_transaction_event-v0.0.1.json",
+    "pid": "1",
+    "creation_date": "2020-01-24T00:00:00",
+    "amount": 2,
+    "type": "fee",
+    "subtype": "overdue",
+    "status": "closed",
+    "location": {
+      "$ref": "https://ils.rero.ch/api/locations/1222"
+    },
+    "parent": {
+      "$ref": "https://ils.rero.ch/api/patron_transaction/1"
+    }
+  },
+  {
+    "$schema": "https://ils.rero.ch/schema/patron_transaction_events/patron_transaction_event-v0.0.1.json",
+    "pid": "2",
+    "creation_date": "2020-01-25T00:00:00",
+    "amount": 0.5,
+    "type": "fee",
+    "subtype": "overdue",
+    "status": "open",
+    "location": {
+      "$ref": "https://ils.rero.ch/api/locations/1222"
+    },
+    "parent": {
+      "$ref": "https://ils.rero.ch/api/patron_transaction/1"
+    }
+  },
+  {
+    "$schema": "https://ils.rero.ch/schema/patron_transaction_events/patron_transaction_event-v0.0.1.json",
+    "pid": "3",
+    "creation_date": "2020-01-25T10:00:00",
+    "amount": 1,
+    "type": "payment",
+    "subtype": "cash",
+    "location": {
+      "$ref": "https://ils.rero.ch/api/locations/1222"
+    },
+    "operator": {
+      "$ref": "https://ils.rero.ch/api/patrons/badr.aly"
```

Comment:
Why `badr.aly` ?

Reply:
That's just a sample data file to explain all events occurred for an `overdue` with "dummy" reference within.
This file will be removed in next PR